lib/getMermaidDiagram.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Generates mermaid class diagram from default syntax
+ * 
+ * @param {Object} metrics Relations between AsyncAPI docs in default syntax
+ * @returns {String} class diagram following mermaid syntax
+ */
+function getMermaidDiagram(metrics) {
+  const visitedChannels = [];
+  const visitedServices = [];
+  let i=1;
+  let mermaidSyntax = '';
+  metrics.forEach((relations,server) => {
+    const url = server.split(',')[0];
+    const protocol = server.split(',')[1];
```

Comment:
You can use here destruction of array syntax, as server after split will be of array type, you can write this line as:

```js
const [url, protocol] = server.split(',');
```

Reply:
Yeah, this way is much better.